src/mimefactory.rs

```diff
@@ -1300,9 +1299,20 @@ impl MimeFactory {
 
         let final_text = placeholdertext.as_deref().unwrap_or(&msg.text);
 
-        let mut quoted_text = msg
-            .quoted_text()
-            .map(|quote| format_flowed_quote(&quote) + "\r\n\r\n");
+        let mut quoted_text = None;
+        if let Some(msg_quoted_text) = msg.quoted_text() {
+            let mut some_quoted_text = String::new();
+            for quoted_line in msg_quoted_text.split('\n') {
+                if !some_quoted_text.is_empty() {
+                    some_quoted_text += "\r\n";
```

Comment:
Can we move this after `some_quoted_text += quoted_line;` and do unconditionally?